app/uk/gov/hmrc/cbcr/models/DocRefID.scala

```diff
@@ -67,11 +70,13 @@ case class DocRefIdRecord(id: DocRefId, valid: Boolean)
 object DocRefIdRecord {
   // Keep in sync with any future frontend changes
   implicit val format: OFormat[DocRefIdRecord] = Json.format[DocRefIdRecord]
-  val dateFmt = DateTimeFormatter.ofPattern("yyyyMMdd'T'HHmmss")
+  @unused
+  val dateFmt: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd'T'HHmmss")
```

Comment:
There's still one left. This one almost definitely can be deleted if it's not used.